content/en/docs/concepts/otel-concepts.md

```diff
@@ -0,0 +1,392 @@
+---
+title: "OTel Concepts"
+weight: 11
+---
+
+This page dives into OpenTelemetry concepts.
+
+## Spans in OpenTelemetry
+
+A [**Span**](/docs/concepts/observability-primer/#spans) represents a unit of
+work or operation. Spans are the building blocks of Traces. In OpenTelemetry,
+they include the following information:
+
+- Name
+- Start and End Timestamps
+- [Span Context](#span-context)
+- [Attributes](#attributes)
+- [Span Events](#span-events)
+- [Span Links](#span-links)
+- [Span Status](#span-status)
+
+Sample Span:
+
+```json
+{
+  "trace_id": "7bba9f33312b3dbb8b2c2c62bb7abe2d",
+  "parent_id": "",
+  "span_id": "086e83747d0e381e",
+  "name": "/v1/sys/health",
+  "start_time": "2021-10-22 16:04:01.209458162 +0000 UTC",
+  "end_time": "2021-10-22 16:04:01.209514132 +0000 UTC",
+  "status_code": "STATUS_CODE_OK",
+  "status_message": "",
+  "attributes": {
+    "net.transport": "IP.TCP",
+    "net.peer.ip": "172.17.0.1",
+    "net.peer.port": "51820",
+    "net.host.ip": "10.177.2.152",
+    "net.host.port": "26040",
+    "http.method": "GET",
+    "http.target": "/v1/sys/health",
+    "http.server_name": "mortar-gateway",
+    "http.route": "/v1/sys/health",
+    "http.user_agent": "Consul Health Check",
+    "http.scheme": "http",
+    "http.host": "10.177.2.152:26040",
+    "http.flavor": "1.1"
+  },
+  "events": {
+    "name": "",
+    "message": "OK",
+    "timestamp": "2021-10-22 16:04:01.209512872 +0000 UTC"
+  }
+}
+```
+
+### Span Context
+
+Span Context provides specific context about the trace and span using two
+identifiers: Trace ID and Span ID. Each Span is identified by a unique ID called
+a Span ID. A Span uses a Trace ID to identify the relationship between span and
```

Comment:
Might need to hammer out the wording here.
1. Span IDs are randomly generated, so there is no guarantee of uniqueness, although the space is big enough so we shouldn't seem collision under the same trace.
2. The Span ID space is not large enough to give a high confidence of no-collision across different traces.

Reply:
Probably unique within a trace? At least in practice that ought to be true

Reply:
Changed to `Each Span is identified by an ID that is unique within a Trace called
a Span ID.`